core/src/main/resources/jenkins/model/Jenkins/login.jelly

```diff
@@ -145,8 +150,9 @@ THE SOFTWARE.
               </j:forEach>
 
               <div class="footer">
-
-                <st:include it="${simpleDecorator}" page="simple-footer.jelly" optional="true"/>
+                <j:forEach var="simpleDecorator" items="${simpleDecorators}">
```

Comment:
This is actually the only change you need besides creating the methods to get all decorators.